modules/using-must-gather.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+[id="using-must-gather_{context}"]
+= Using Master-Gather
+
+include::modules/common-attributes.adoc[]
+:context: migrating-3-4
+:migrating-3-4:
+
+Must-Gather for MTC is a tool built on top of {product-title}’s standard must-gather tool to gather MTC-specific resources. Therefore, you need to run Must-Gather for MTC  when you migrate containers.
+
+.Prerequisites
+
+* {product-title} 4.x
```

Comment:
Too brief. Explain what the prerequisite is and why

Reply:
Fixed.